server/checks/PullRequestTasks.js

```diff
@@ -0,0 +1,82 @@
+import Check, { getPayloadFn } from './Check'
+import { PULL_REQUEST } from '../model/GithubEvents'
+import { logger } from '../../common/debug'
+
+const CHECK_TYPE = 'pullrequesttasks'
+const CONTEXT = 'zappr/pr/tasks'
+const info = logger(CHECK_TYPE, 'info')
+const error = logger(CHECK_TYPE, 'error')
+const debug = logger(CHECK_TYPE)
+const createStatePayload = getPayloadFn(CONTEXT)
+
+export function hasOpenTasks(pr) {
```

Comment:
Hm, got it. I really don't like passing a full-fledged PR object into a function that's supposed to work on a string. Can we add repoName and prNo as optional parameters?

~~~ javascript
function hasOpenTasks(prBody, repoName = '', prNumber = '') {
...
}
~~~